File: Assets/Scripts/StartButtonController.js
#pragma strict

var texture : Texture;

function OnGUI() {
	var origin = new Vector2(Screen.width / 2 + 150, Screen.height / 2 - 150);
	var size = new Vector2(250, 60);
	
	if (GUI.Button(Rect(origin.x,origin.y,size.x,size.y),texture))
        GameController.instance.StartGame();
}

File: Assets/Scripts/StartButtonController.cs
using UnityEngine;
using System.Collections;

public class StartButtonController : MonoBehaviour {
	public Texture texture;

	public void OnGUI() {
		Vector2 origin = new Vector2(Screen.width / 2 + 150, Screen.height / 2 - 150);
		Vector2 size = new Vector2(250, 60);
		
		if (GUI.Button(new Rect(origin.x,origin.y,size.x,size.y),texture))
	        GameController.instance.StartGame();
	}
}
